src/index.html

```diff
@@ -6,9 +6,53 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+
+    <header>
+      <div class="container">
+        <div class="logo">
+          <a href="#home">
+            <img 
+              src="/src/images/logo.png"
+              alt="#home"
+              href="#back-home" 
+              class="photo"
+            >
+          </a>        
+        </div>
+  
+        <nav class="navigation">
+          <ul class="nav_list">
+            <li class="nav-item">
```

Comment:
```suggestion
            <li class="nav_item">
```
Назви класів мають бути в одному стилі. Якщо ти скріщт використовуєш нижнє підкреслювання, то і тут потрібно

Reply:
ок